lib/linguist/languages.yml

```diff
@@ -4847,6 +4847,17 @@ POV-Ray SDL:
   tm_scope: source.pov-ray sdl
   ace_mode: text
   language_id: 275
+Pact:
+  type: programming
+  color: "#F7A8B8"
+  ace_mode: text
+  tm_scope: source.pact
+  extensions:
+  - ".pact"
+  - ".repl"
```

Comment:
I've looked into this a bit more closely now and there are a lot of other users of this extension which don't appear to be Pact from my naïve perspective. I'm not sure what all those other languages are but they seem to outweigh the Pact files so it's probably best to remove this extension and sample for now. Peeps can implement an override to force the language once this PR has been merged.

Reply:
That sounds okay to me. There is indeed quite a bit of overlap with other languages, and I'm happy to just sit with `.pact` highlighting

Reply:
Updated!